exporter/signalfxexporter/translation/delta_translator.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/gogo/protobuf/proto"
 	sfxpb "github.com/signalfx/com_signalfx_metrics_protobuf/model"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter/ttlmap"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ttlmap"
```

Comment:
I suggest that we move to `internal/common` instead of `pkg`. I do not want someone else to depend on us and then us breaking them accidentally if we need to make changes.

Reply:
Done!